src/main/java/com/google/api/codegen/transformer/StaticLangApiMethodTransformer.java

```diff
@@ -61,7 +61,36 @@
   private final SampleTransformer sampleTransformer;
 
   public StaticLangApiMethodTransformer(SampleType sampleType) {
-    this.sampleTransformer = new SampleTransformer(sampleType);
+    this.initCodeTransformer = new InitCodeTransformer();
+    this.sampleTransformer = SampleTransformer.create(sampleType);
+  }
+
+  public StaticLangApiMethodTransformer(
```

Comment:
A possible refactoring here would be to have only one new constructor, that accepted a `SampleTransformer`, instead of multiple constructors that take combinations of parameters used to create the `SampleTransformer`. WDYT? The same applies to the DynamicLangApiMethodTransformer constructors too.